client/state/themes/actions.js

```diff
@@ -75,26 +75,26 @@ export function incrementThemesPage( site ) {
 	};
 }
 
-export function fetchCurrentTheme( siteId ) {
+export function requestActiveTheme( siteId ) {
```

Comment:
Can we have a JSDoc block? 😄 Just once sentence should be enough (plus `@param` and `@return`).